src/app/_shared/components/yt-video-player/yt-video-player.component.ts

```diff
@@ -0,0 +1,25 @@
+import { Component, Input, OnInit, ViewEncapsulation } from '@angular/core';
+
+@Component({
+  selector: 'app-yt-video-player',
+  templateUrl: './yt-video-player.component.html',
+  styleUrls: ['./yt-video-player.component.scss'],
+  host: {
+    'class': 'app-yt-video-player'
+  },
+  encapsulation: ViewEncapsulation.None
+})
+export class YtVideoPlayerComponent implements OnInit {
+
+  @Input() videoId: string | undefined = undefined;
+
+  playerVars: YT.PlayerVars = {
+    rel: YT.RelatedVideos.Hide
+  };
+
+  constructor() { }
+
+  ngOnInit(): void {
+  }
```

Comment:
I would remove empty `constructor` and `ngOnInit` . There is an eslint rule that errors if we keep empty methods. 
I think there is an issue opened on migrating from ts-lint to es-lint

Reply:
Done  😉